FIPS/pom.xml

```diff
@@ -38,17 +38,17 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <arrow.version>10.0.1</arrow.version>
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.6</slf4j.version>
     <jsoup.version>1.15.3</jsoup.version>
     <tika.version>2.4.1</tika.version>
     <jackson.version>2.14.2</jackson.version>
-    <jacksondatabind.version>2.13.4.2</jacksondatabind.version>
```

Comment:
why remove this?

Reply:
jacksondatabind becomes the same version as jackson.version